src/plugins/sentry.ts

```diff
@@ -2,20 +2,20 @@ import { App } from 'vue';
 import { captureException, init, setTag } from '@sentry/browser';
 import { Integrations } from '@sentry/tracing';
 import { version } from '../../package.json';
+import { networkId } from '@/composables/useNetwork';
 
 // Using Sentry's vanila JS package (@sentry/browser) here instead of
 // the official vue package (@sentry/vue) because it doesn't support vue 3 yet.
 // https://github.com/getsentry/sentry-javascript/issues/2925
 
 const ENV = process.env.VUE_APP_ENV || 'development';
-const NETWORK = process.env.VUE_APP_NETWORK || '42';
 const networkMap = {
   '1': 'mainnet',
   '42': 'kovan',
   '137': 'polygon',
   '42161': 'arbitrum-one'
 };
-const environment = `${ENV}-${networkMap[NETWORK]}`;
+const environment = `${ENV}-${networkMap[networkId.value]}`;
```

Comment:
Will this work? The `networkMap` keys are strings but I think the `networkId.value` is a number right? This is a more general issue. There are a lot of places where the networkId is expected to be a string and not a number.

Reply:
Ah, nice catch. I'll fix this. (change to number)

`networksMap[1] === networksMap['1']` because numeric keys are always converted to strings.